app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,73 @@
+class TweetController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?
+      @user = current_user
+      @tweets = Tweet.all
+      erb :"views/tweets/tweets"
+     else
+       redirect to "/login"
+     end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      @user = current_user
+      erb :'views/tweets/create_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/new/tweet' do
+    if !params[:tweet][:content].blank?
+      tweet =Tweet.create(params[:tweet])
+      tweet.save
+      redirect to '/tweets'
+    else
+      redirect to '/tweets/new'
+    end
+  end
+
+  get '/tweets/:id' do
+    if logged_in?
+      @tweet = Tweet.find(params[:id])
+      erb :'views/tweets/show_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/delete/:id' do
```

Comment:
Deleting a tweet would normally be a delete request to the `'/tweets/:id'` route.